forc/src/ops/forc_update.rs

```diff
@@ -32,14 +33,23 @@ pub async fn update(command: UpdateCommand) -> Result<()> {
         None => std::env::current_dir()?,
     };
 
-    let manifest = PackageManifestFile::from_dir(&this_dir)?;
+    let manifest = ManifestFile::from_dir(&this_dir)?;
     let lock_path = lock_path(manifest.dir());
     let old_lock = Lock::from_path(&lock_path).ok().unwrap_or_default();
     let offline = false;
-    let new_plan = pkg::BuildPlan::from_manifest(&manifest, offline)?;
+    let member_manifests = manifest.member_manifests()?;
+    let new_plan = pkg::BuildPlan::from_manifests(&member_manifests, offline)?;
     let new_lock = Lock::from_graph(new_plan.graph());
     let diff = new_lock.diff(&old_lock);
-    lock::print_diff(&manifest.project.name, &diff);
+    let member_names = if let ManifestFile::Package(pkg_manifest) = &manifest {
+        HashSet::from([pkg_manifest.project.name.clone()])
+    } else {
+        member_manifests
+            .iter()
+            .map(|(_, manifest)| manifest.project.name.clone())
+            .collect()
+    };
```

Comment:
We don't really need to check if `manifest` is `ManifestFile::Package` here (it will already exist in `member_manifests`). We can change this to:
```rust
let member_names = member_manifests
    .iter()
    .map(|(_, manifest)| manifest.project.name.clone())
    .collect();
```